src/static/mixpanelEventId.tsx

```diff
@@ -0,0 +1,5 @@
+export const EVENT_ID = {
+  CLICKED_INTERRUPT: "interrupt_clicked",
+  CLICKED_INFORMATION: "information_clicked",
+  CLICKED_FORM: "form_clicked",
```

Comment:
I recommend you to name the same as 'CATEGORY_CLICKED' and add a [mixpanel filter.](https://docs.mixpanel.com/docs/quickstart/capture-events/track-events)

Reply:
I was going to do that, but I think the mixpanel dashboard is rather less readable. 

In the way you told me, the first screen on the dashboard will look like only events named CATEGORY_CLICKED are registered. Of course, we can see what kind of events they are in detail by clicking on the details. 

There aren't many events to track at the moment, and I think it's a better way to give an event name for each click in this situation.

What do you think?